docs/UsingOmnisci/index.rst

```diff
@@ -22,5 +22,17 @@ or turn it on in source code:
    cfg.StorageFormat.put('omnisci')
    cfg.IsExperimental.put(True)
 
+To enable ``OmniSci`` engine launch export :
```

Comment:
This phrase along with the phrase above is a bit confusing to me and might be confusing for other users.
```
1. To enable this engine you could set the following environment variables
2. To enable ``OmniSci`` engine launch export :
```
I would corrected second line like this:
```
There is a more simple way to enable ``OmniSci`` engine. You can only export `MODIN_STORAGE_FORMAT` environment variable and Modin itself defines the rest ones:
```